torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1065,6 +1078,7 @@ def test_py_tensors(self):
         self.assertEqual(ret, my_tensor_function(torch.ones(n, n), torch.ones(n, n)))
 
     @dist_init
+    @_skip_if_tensorpipe_agent
```

Comment:
In the previous test the `_skip_if_tensorpipe_agent` came before the `dist_init` decorator, here it comes after. Does it make any difference? You mentioned in the PR's summary that `dist_init` was setting some value to processgroup as default, perhaps it's due to this?